### ISSUE TITLE
Empty rootName prop now displays <empty string>

### ISSUE THREAD

**Issue by borzaka** [CarlosNZ/json-edit-react#69]
**Describe the bug**
In the 1.11.0 version I was able to hide the `rootName`, because I don't need for aesthetic reasons. I wanted to make the editor as clean as possible.
With the 1.11.7 version it's no longer possible. I'am forced to use some `rootName`, otherwise an `<empty string>` string will be displayed, which is not so beautiful.

**Expected behavior**
Empty `rootName` prop should hide the `rootName` as in prev versions.

**Screenshots**
v1.11.0
![image](https://github.com/CarlosNZ/json-edit-react/assets/1230402/f13bffe7-278f-44af-b3b3-3a5e387bb712)

v1.11.7
![image](https://github.com/CarlosNZ/json-edit-react/assets/1230402/af928342-e0d3-457e-b5d6-3a845b5f8dd5)

My code:
```typescript
<JsonEditor
  data={JSON.parse(configProperty.metaValue as string)}
  rootName=""
  enableClipboard={false}
  indent={2}
  restrictEdit={true}
  restrictDelete={true}
  restrictAdd={true}
  showCollectionCount={false}
  maxWidth="100%"
  className="form-control rounded-0 mb-3"
/>
```

**Comment by CarlosNZ** [comment]
Ah yes, sorry about that. This was due to the fix I did for https://github.com/CarlosNZ/json-edit-react/issues/66. 

I'll make a quick fix for this very soon.

In the meantime you can actually override it with the by setting `content: ""` on the CSS class `.jer-empty-string::after`, but I'll sort it out.

Thanks for bring it to my attention.

**Comment by CarlosNZ** [comment]
Okay, this should be fixed now via #70.  Available now in [`v1.11.8`](https://www.npmjs.com/package/json-edit-react/v/1.11.8)

Apologies for the inconvenience, thanks again for pointing out.
